doc/users/prev_whats_new/whats_new_3.6.0.rst

```diff
@@ -147,6 +150,8 @@ The new *gapcolor* parameter to `~.Axes.plot` enables the creation of striped
 lines.
 
 .. plot::
+    :alt: Plot of x**3 where the line is an orange-blue striped line, achieved using the keywords`linestyle='--', color='orange', gapcolor='blue' 
+    
```

Comment:
```suggestion
```

This blank line is causing this plot directive to treat `:include-source: true` as python code, which is a syntax error

Reply:
Thank you!